index.html

```diff
@@ -60,66 +72,1830 @@
+                </div>
+            </div>
+            <div class="col-md-4">
+                <article class="mb-3 py-2">
+                    <div class="float-start">
+                       <a href="#"><img src="img/expert.png" class="img-fluid swidth me-3 pt-2" alt="..."></a>
+                    </div>
+                    <div class="clearfix">
+                        <h5><a href="#" class="link-dark nav-link p-0 mt-2 fw-bold font-size">'Tiger King' Zoo Owners Ordered To Hand Over Big Cats to Government</a></h5>
+                    </div>
+                </article>
+                <article class="border-top border-2 mb-3 py-2">
+                    <div class="float-start">
+                       <a href="#"><img src="img/expert.png" class="img-fluid swidth me-3 pt-2" alt="..."></a>
+                    </div>
+                    <div class="clearfix">
+                        <h5><a href="#" class="link-dark nav-link p-0 mt-2 fw-bold font-size">'Tiger King' Zoo Owners Ordered To Hand Over Big Cats to Government</a></h5>
+                    </div>
+                </article>
+                <article class="border-top border-2 mb-3 py-2">
+                    <div class="float-start">
+                       <a href="#"><img src="img/expert.png" class="img-fluid swidth me-3 pt-2" alt="..."></a>
+                    </div>
+                    <div class="clearfix">
+                        <h5><a href="#" class="link-dark nav-link p-0 mt-2 fw-bold font-size">'Tiger King' Zoo Owners Ordered To Hand Over Big Cats to Government</a></h5>
+                    </div>
+                </article>
+            </div>
+        </section>
+        <div class="gray w-100 d-md-none"> <br></div>
+        <div class="row ">
+         <!--Subscribe-->
+        <section class="my-3 mx-2 border-1 col-lg-7">
+            <header><h5 class="otext fw-bold">SUBSCRIBE</h5></header>
+            <div class="bg-light jumbotron mt-3 py-5">
+                <div class="d-flex flex-column align-items-center flex-md-row align-items-md-end">
+                    <div class="position-relative p-3">
+                       
+                        <div class="subdiv2"></div>
+                    </div>
+                    <div class="text-box d-flex flex-column align-items-center align-items-md-start p-4">
+                        <strong class="fs-4">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</strong>
+                        <p>All Access + Weekly Delivery.
+                            Choose A Membership That's Perfect for You!</p>
+                        <button class="btn bg-news text-white fw-bold">SUBSCRIBE &gt;</button>
+                    </div>
+                </div>  
+            </div>                        
+        </section>
+        <div class="gray w-100 d-md-none"> <br></div>
```

Comment:
- [ ] subscribe section should have a clickable button. 
- [ ] The text is hidden. 
### original 👇 
![image](https://user-images.githubusercontent.com/64360974/114854458-85a62400-9ded-11eb-96bf-5d0d24e0057b.png)

### Yours 👇 
![image](https://user-images.githubusercontent.com/64360974/114854423-7a52f880-9ded-11eb-8b68-ba84a4b43198.png)